--- a/libobs/CMakeLists.txt
+++ b/libobs/CMakeLists.txt
@@ -1,4 +1,6 @@
 project(libobs)
+
+list(APPEND CMAKE_MODULE_PATH "/usr/share/ECM/find-modules")
 
 find_package(Threads REQUIRED)
 
--- a/plugins/linux-pulseaudio/CMakeLists.txt
+++ b/plugins/linux-pulseaudio/CMakeLists.txt
@@ -1,4 +1,6 @@
 project(linux-pulseaudio)
+
+list(APPEND CMAKE_MODULE_PATH "/usr/share/ECM/find-modules")
 
 if(DISABLE_PULSEAUDIO)
 	message(STATUS "PulseAudio support disabled")

