--- a/libobs/obs-nix.c	2021-01-26 00:31:31.853462587 -0500
+++ b/libobs/obs-nix.c	2021-01-26 00:35:18.229645660 -0500
@@ -54,11 +54,13 @@
 
 static const char *module_bin[] = {"../../obs-plugins/" BIT_STRING,
 				   OBS_INSTALL_PREFIX
-				   "/" OBS_PLUGIN_DESTINATION};
+				   "/" OBS_PLUGIN_DESTINATION,
+				   "/app/plugins/lib/obs-plugins"};
 
 static const char *module_data[] = {
 	OBS_DATA_PATH "/obs-plugins/%module%",
 	OBS_INSTALL_DATA_PATH "/obs-plugins/%module%",
+	"/app/plugins/share/obs/obs-plugins/%module%",
 };
 
 static const int module_patterns_size =
